### PR TITLE
Fixed the drop down not getting closed when add contact

### PR DESCRIPTION
**Pull request by mritunjay51**

### CODE REVIEW

File: src/foam/u2/view/RichChoiceView.js [hunk 1]
```diff
@@ -427,6 +427,12 @@ foam.CLASS({
         return this.CHOOSE_FROM + ' ' + plural + '...';
       }
     },
+    {
+      type: 'Boolean',
```
Comment (kgrgreer) [inline src/foam/u2/view/RichChoiceView.js]: Should be class: 'Boolean'

File: src/foam/u2/view/RichChoiceView.js [hunk 2]
```diff
@@ -427,6 +427,12 @@ foam.CLASS({
...
+      name: 'isExternalAction',
+      documentation: `Flag to check if newly added element is outside dropdown`,
+      value: false
```
Comment (kgrgreer) [inline src/foam/u2/view/RichChoiceView.js]: Not needed since false is the default value by default.

File: src/foam/u2/view/RichChoiceView.js [hunk 3]
```diff
@@ -628,13 +634,23 @@ foam.CLASS({
       if ( action && actionData ) {
         return this.E()
           .start(self.DefaultActionView, { action: action, data: actionData })
-            .addClass(self.myClass('action'))
+          .callIf(self.isExternalAction == true, function () {
```
Comment (kgrgreer) [inline src/foam/u2/view/RichChoiceView.js]: "== true" is redundant

File: src/foam/u2/view/RichChoiceView.js [hunk 4]
```diff
@@ -628,13 +634,23 @@ foam.CLASS({
...
+            });
+          })
+          .addClass(self.myClass('action'))
           .end();
       }
       if ( action ) {
         return this.E()
           .start(self.DefaultActionView, { action: action })
-            .addClass(self.myClass('action'))
+          .callIf(self.isExternalAction == true, function () {
```
Comment (sarthak-marwaha) [inline src/foam/u2/view/RichChoiceView.js:648]: Do we need this at all? Dont we want it to always close on click of the action?

File: src/foam/u2/view/RichChoiceView.js [hunk 5]
```diff
@@ -628,13 +634,23 @@ foam.CLASS({
...
+            this.on('click', () => {
+              self.dropdown_.close();
+            });
+          })
```
Comment (sarthak-marwaha) [inline src/foam/u2/view/RichChoiceView.js]: Do we need this at all? Dont we want it to always close on click of the action? Alternative below
```suggestion
          .on('click', () => {
              self.dropdown_.close();
            })
```

**Pull request merged**